wagtail/admin/templates/wagtailadmin/shared/side_panels/status.html

```diff
@@ -16,7 +16,11 @@
         </div>
         {% if object.first_published_at %}
             <div class="w-help-text">
-                {% trans 'First published ' %}{% timesince_last_update object.first_published_at use_shorthand=True %}
+                <button type="button" class="w-human-readable-date" data-tippy-content="{{ object.first_published_at|date:'M. d, Y, H:i:s' }}" data-tippy-placement="bottom">
+                    <time class="w-human-readable-date__date" datetime="{{ object.first_published_at }}">
+                        {% trans 'First published ' %}{% timesince_last_update object.first_published_at use_shorthand=True %}
+                    </time>
```

Comment:
Is it possible to use the template tag instead - e.g. 

```
{% human_readable_date my_model.timestamp %}
```

Search around the code, we have used this everywhere where this kind of UI is needed. 

Here's the code for that tag. 

https://github.com/wagtail/wagtail/blob/main/wagtail/admin/templatetags/wagtailadmin_tags.py#L1198

Reply:
@lb- thanks for the response. I'll look into it and update it accordingly.

Reply:
Hello @lb- Sir,

I've tried to update the code as per your suggestion to use the template tag. However, I'm facing two issues:

1. Difficulty displaying the tooltip at the bottom.
2. "First Published" is not showing tooltip on hover.

**Updated Code:**

`{% if object.first_published_at %}
    <div class="w-help-text">
        {% trans 'First published' %} {% human_readable_date object.first_published_at %}
    </div>
{% endif %}`

I've attached a screenshot for reference. Could you please provide further guidance on these issues?

![Screenshot (24)](https://github.com/wagtail/wagtail/assets/111359305/606b88ab-fe0b-46ca-ad2c-34b59010bf42)

Reply:
A simple solution might be to move the page type label to under the published date. 

Alternatively, you can update the template tag to accept a position arg, then pass / use this in the template.

Reply:
Thanks @lb-  for your guidance. I've implemented the alternative method you suggested, and now the tooltip is functioning correctly. Below, I'm sharing the updated code for the template tag as well as a screenshot. Please review them and let me know if they look good to proceed.

**Template tag**

![Screenshot (25)](https://github.com/wagtail/wagtail/assets/111359305/3d353413-f701-4fca-9598-1adb2da16fd8)

**Human_readable_date.html**

```
{% load wagtailadmin_tags i18n %}

<button type="button" class="w-human-readable-date" data-tippy-content="{{ date|date:"DATETIME_FORMAT" }}" data-tippy-placement="{{ position }}">
    <time class="w-human-readable-date__date" datetime="{{ date|date:"c" }}">
        {{ date|timesince_simple|capfirst }}
    </time>
    {% if description %}
        <span class="w-human-readable-date__description">{{ description }}</span>
    {% endif %}
</button>
```

**status.html (updated code)**

```
{% if object.first_published_at %}
    <div class="w-help-text">
        {% trans 'First published' %} {% human_readable_date object.first_published_at position="bottom" %}
    </div>
{% endif %}
```
**Screenshot**

![Screenshot (26)](https://github.com/wagtail/wagtail/assets/111359305/eb1f29ea-9407-4f57-ab06-ee7026d098c1)